lighthouse-core/scripts/release/test.sh

```diff
@@ -39,7 +36,7 @@ cd /tmp/lighthouse-local-test
 
 npm init -y
 npm install "$LH_PRISTINE_ROOT/lighthouse-$VERSION.tgz"
-npm explore lighthouse -- npm run smoke
+npm explore lighthouse -- npm run smoke || npm explore lighthouse -- npm run smoke
```

Comment:
and it's nice to run this twice just in case there's some flake (like I just experienced)

Reply:
should we make it three like we just removed above? :)

Reply:
wanna lift the bash-ism i used in #9766 ?

Reply:
FWIW connor that actually required significant cognitive overhead for me to verify it's doing what we want it to do. when it's only a couple times, I'd greatly prefer `|| <repeat>`